folly/experimental/io/test/AsyncIOTest.cpp

```diff
@@ -90,7 +90,7 @@ class TemporaryFile {
 };
 
 TemporaryFile::TemporaryFile(size_t size)
-    : path_(fs::temp_directory_path() / fs::unique_path()) {
+    : path_("/var/tmp" / fs::unique_path()) {
```

Comment:
From @nbronson,

> Rather than hard-coding paths here, maybe we should probe `fs::temp_directory_path()` with an `open` in `O_DIRECT` mode, and then just skip the test if that fails.

For skipping the test, use `SKIP` in `folly/test/TestUtils.h`.

Reply:
This is much better, thanks!